dev-docs/bidders/zeta_global_ssp.md

```diff
@@ -25,4 +25,5 @@ All references to the OpenRTB spec refer to OpenRtb v2.5 (https://www.iab.com/wp
 | `tags`               | optional | The object containing set of  Zeta's custom tags witch publisher have to supply                                     | `tags: {}`   | `object`  |
 | `site`               | optional | The object containing site data (See OpenRTB spec)                                                                  | `site: {}`   | `object`  |
 | `app`                | optional | The object containing app data (See OpenRTB spec)                                                                   | `app: {}`    | `object`  |
+| `user.ext.eids`      | optional | The object provided by SharedId module and contains OpenRTB eids objects                                            | `eids: [{}]` | `object`  |
```

Comment:
I don't think your adapter supports defining `eids` through params?

You need to add supported `userIds` at the top of this .md file. Something like:

```
userIds: britepoolId, criteo, fabrickId, haloId, id5Id, identityLink, idxId, intentIqId, liveIntentId, lotamePanoramaId, merkleId, netId, parrableId, pubCommonId, quantcastId, sharedId, tapadId, unifiedId, verizonMediaId, zeotapIdPlus
```

I took this example from some other bid adapter that is using all userIds.

Reply:
Sorry, fixed it.
Thanks!